### PR TITLE
PMM-7296: Add ListAlertRules pagination tests

### PR DESCRIPTION
**Pull request by oter**
https://jira.percona.com/browse/PMM-7296

FB: https://github.com/Percona-Lab/pmm-submodules/pull/1457

API PR: https://github.com/percona/pmm/pull/662
https://github.com/percona/pmm-managed/pull/642

### CODE REVIEW

**Review by Dasio — APPROVED** [review]

File: management/ia/rules_test.go [hunk 1]
```diff
@@ -359,7 +359,7 @@ func TestRulesAPI(t *testing.T) {
 		}
 	})
 
-	t.Run("list", func(t *testing.T) {
+	t.Run("list without pagination", func(t *testing.T) {
```
Comment (BupycHuk) [inline management/ia/rules_test.go:362]: Does that mean that without pagination we will return all alertRules?

**Review by BupycHuk — APPROVED** [review]

File: management/ia/rules_test.go (hunk 1 above)
Comment (oter) [reply management/ia/rules_test.go:362]: no, just the first page with default page size 
https://github.com/percona/pmm-managed/pull/642/files#diff-3579dfaf2503ed8fb00f2bfe06ee104c8c5bbb9d89c3a59fbd7afa52ecf3ed46R259

**Review by artemgavrilov — APPROVED** [review]

File: management/ia/rules_test.go [hunk 2]
```diff
@@ -398,6 +398,76 @@ func TestRulesAPI(t *testing.T) {
...
+		// some tests didn't remove rules, so expect more elements than created in current test
+		assert.GreaterOrEqual(t, len(lp1.Rules), rulesCount)
+		assert.Equal(t, int32(len(lp1.Rules)), lp1.Totals.TotalItems)
+		assert.Equal(t, int32(1), lp1.Totals.TotalPages)
+		for id := range ruleIDs {
+			var found bool
+			for _, r := range list1.Payload.Rules {
+				if r.RuleID == id {
+					found = true
+					break
```
[the remark this answers is not in the document]
Comment (oter) [reply management/ia/rules_test.go:441]: fixed

File: management/ia/rules_test.go (hunk 1 above)
Comment (oter) [reply management/ia/rules_test.go:362]: resolving

**Pull request merged**